clients/lambda.d.ts

```diff
@@ -328,8 +329,8 @@ declare namespace Lambda {
   export type AliasList = AliasConfiguration[];
   export type Arn = string;
   export type BatchSize = number;
-  export type _Blob = Buffer|Uint8Array|Blob|string;
-  export type BlobStream = Buffer|Uint8Array|Blob|string;
+  export type _Blob = Buffer|Uint8Array|Blob|string|Readable;
```

Comment:
I don't believe this shape is streaming (see https://github.com/aws/aws-sdk-js/blob/v2.94.0/apis/lambda-2015-03-31.normal.json#L646 )